src/Dns.cs

```diff
@@ -6,41 +6,49 @@
 using ImportsWorld;
 using ImportsWorld.wit.imports.wasi.sockets.v0_2_1;
 
-public class Dns
+namespace Wasi.Tls
 {
-    public static async Task<IPAddress[]> GetHostAddressesAsync(string name)
+    public class Dns
     {
-        if (IPAddress.TryParse(name, out IPAddress? parsed))
+
+        public static IPAddress[] GetHostAddresses(string hostNameOrAddress)
         {
-            return new IPAddress[] { parsed };
+            return GetHostAddressesAsync(hostNameOrAddress).Result;
```

Comment:
Does this work in general?  I'm surprised we don't need to run the `WasiEventLoop` here.

Reply:
it has been so far

Reply:
I haven't tried with an actual dns look up

Reply:
Ah, ok.  Yeah, I think we'll need to use WasiEventLoop for actual DNS lookups.